msteams-platform/resources/schema/manifest-schema-dev-preview.md

```diff
@@ -991,6 +991,31 @@ The `extensions.runtimes` array configures the sets of runtimes and actions that
 |`actions.view`| String | 64 characters | | Specifies the view where the page must be opened. It's used only when `actions.type` is `openPage`. |
 |`actions.multiselect`| Boolean | | | Specifies whether the end user can select multiple items, such as multiple email messages, and apply the action to all of them.|
 |`actions.supportsNoItemContext`| Boolean | | | Allows task pane add-ins to activate without the Reading Pane enabled or a message selected. |
+|`customFunctions`| Object | | | Custom functions enable developers to add new functions to Excel by defining those functions in JavaScript as part of an add-in. Users within Excel can access custom functions just as they would any native function in Excel, such as **SUM**. |
+|`customFunctions.functions`| Array | 20,000 | ✔️ | An array of custom function objects that define custom functions metadata. |
+|`customFunctions.functions.id`| String | 64 characters | ✔️ | Specifies a unique ID for the function. This ID can only contain alphanumeric characters and periods and shouldn't be changed after it's set. It must start with a letter and have a minimum of three characters. |
+|`customFunctions.functions.name`| String | 64 characters | ✔️ | The name of the function that end users see in Excel. In Excel, this function name is prefixed by the custom functions namespace that is specified in the manifest file. It's unique within the scope of the file and can only contain alphanumeric characters. It must start with a letter and have a minimum of three characters. It shouldn't be the same as any cells between **A1** and **XFD1048576**, any cells between **R1C1** and **R1048576C16384**, or any Excel 4.0 macro function (such as **RUN** or **ECHO**).|
```

Comment:
If it's a requirement or a convention that this value be ALL CAPS, we should probably say that here.

Reply:
Added "Use uppercase letters as a best practice." to customFunctions.functions.id and customFunnctions.functions.name.